coordinator/history/history_test.go

```diff
@@ -59,18 +61,20 @@ func TestHistory_GetLatestAndHasLatest(t *testing.T) {
 			wantHasLatest: true,
 		},
 		"no latest": {
-			fsContent:     map[string]string{},
-			signingKey:    testkeys.New[ecdsa.PrivateKey](t, testkeys.ECDSAP256Keys[0]),
-			wantErr:       true,
-			wantHasLatest: false,
+			fsContent:       map[string]string{},
+			signingKey:      testkeys.New[ecdsa.PrivateKey](t, testkeys.ECDSAP256Keys[0]),
+			wantErr:         true,
+			wantHasLatest:   false,
+			wantInsecureErr: true,
 		},
 		"no signature": {
 			fsContent: map[string]string{
 				"transitions/latest": fromHex(rq, "2cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b9824"),
 			},
-			signingKey:    testkeys.New[ecdsa.PrivateKey](t, testkeys.ECDSAP256Keys[0]),
-			wantErr:       true,
-			wantHasLatest: true,
+			signingKey:      testkeys.New[ecdsa.PrivateKey](t, testkeys.ECDSAP256Keys[0]),
+			wantErr:         true,
+			wantHasLatest:   true,
+			wantInsecureErr: true,
```

Comment:
Shouldn't we have new case where `GetLatestInsecure` succeeds but `GetLatest` donesn't?

Reply:
That should be the case in `signature modified` above.

Reply:
makes sense.